tools/pipelines/build-azure-local-service.yml

```diff
@@ -21,6 +21,10 @@ parameters:
     - default
     - skip
     - force
+- name: buildToolsVersionToInstall
+  displayName: Fluid build tools version (default = installs version in repo)
```

Comment:
should it default to not repo?  the repo version does add non-trivial time to the pipeline.

Reply:
Once the build tools are baked, I'll update the default for main. In lts it'll default to npm, but main for now will use the in-repo version.